sw/device/silicon_creator/lib/sigverify/sphincsplus/params.h

```diff
@@ -0,0 +1,107 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Derived from code in the SPHINCS+ reference implementation (CC0 license):
+// https://github.com/sphincs/sphincsplus/blob/ed15dd78658f63288c7492c00260d86154b84637/ref/params/params-sphincs-shake-128s.h
```

Comment:
@arunthomas @mundaym @moidx @cfrantz any comments on this license header?

Reply:
CC0 is one of the most permissive licenses; the author even waives copyright on their work and dedicates it to the public domain for free use and modification. In a case like this where I've made some significant modifications to the base code, I believe this form of credit is enough (even this is not technically required by the license terms, but it seems like good manners).

Reply:
SG, thanks!